src/EditorFeatures/TestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -72,12 +72,18 @@ public static async Task<RemoteHostClient> CreateAsync(HostWorkspaceServices ser
         /// Remote API.
         /// </summary>
         public Task GetAssetsAsync(int scopeId, Checksum[] checksums, string pipeName, CancellationToken cancellationToken)
-            => RemoteEndPoint.WriteDataToNamedPipeAsync(
+        {
+            var remotableDataService = _services.GetRequiredService<IRemotableDataService>();
+            var storage = RemoteHostAssetSerialization.TryGetStorage(remotableDataService, scopeId);
```

Comment:
super confused what thsi scopeId is referring to. is it a field?

Reply:
I believe it's the key for a PinnedRemotableDataScope, and used to identify a storage context which is valid for the duration of a single call from devenv&rarr;OOP. @tmat to confirm

Reply:
➡️ Consolidating this thread with https://github.com/dotnet/roslyn/pull/45953#discussion_r454109186